core-web/libs/sdk/uve/src/lib/types/editor/public.ts

```diff
@@ -62,3 +62,12 @@ export type UVESubscription = {
  * @typedef {function} UVEEvent
  */
 export type UVEEvent = (callback: UVECallback) => UVESubscription;
+
+/**
+ * Configuration type for DotCMS Editor
+ * @typedef {Object} DotCMSEditoConfig
+ * @property {Object} [params] - Parameters for Page API configuration
+ * @property {number} [params.depth] - The depth level for fetching page data
+ * @property {string} [query] - GraphQL query string for data fetching
+ */
+export type DotCMSEditoConfig = { params: { depth: number } } | { query: string };
```

Comment:
Looks like we can use here the `DotCMSClientParams` type and maybe add another one for the `query` variant

Reply:
We'll deprecate this config, @zJaaal. We should a config similar to the new `client.page.get()`